ethereum/VERIFY.md

```diff
@@ -8,7 +8,7 @@ transaction payloads according to the contract ABI.
 
 This document outlines the process of verification. In general, you will need an
 API key for the relevant explorer (this can be obtained by creating an account),
-and also know which address the contract code lives. The API key is expected to
+and also knowing which address the contract code is located. The API key is expected to
```

Comment:
I think it should be "knowing at which address..."

Reply:
Thanks for the input. I'll tweak it to "knowing at which address".